.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["2.4", "2.5", "2.6", "2.7"]
+        ruby-version: ["2.4", "2.5", "2.6", "2.7", "3.0"]
```

Comment:
Thanks for adding the 3.0!

Perhaps we can order the list in 3.0-2.x order?

Reply:
yes, sure.. pushed it.